modules/apps/web-experience/application-list/application-list-api/src/main/java/com/liferay/application/list/BasePanelApp.java

```diff
@@ -42,6 +42,15 @@
 import javax.servlet.http.HttpServletResponse;
 
 /**
+ * This class provides a skeletal implementation of the {@link PanelApp} in
+ * order to minimize the effort required to implement this interface.
+ *
+ * To implement an application the programmer needs to extend this class and
+ * override <code>include</code> method which returns <code>true</code> in case
+ * of successful rendering of application view and <code>false</code> otherwise.
```

Comment:
@pavel-savinov are we suggesting that the developer should override the `BasePanelApp.include` method in their own extending class to return `true` for successful rendering and `false` otherwise?